src/main/java/com/frameworkium/jira/zapi/SearchExecutions.java

```diff
@@ -1,30 +1,28 @@
 package com.frameworkium.jira.zapi;
 
 import static com.frameworkium.config.SystemProperty.JIRA_URL;
-import static com.jayway.restassured.RestAssured.get;
-import static com.jayway.restassured.RestAssured.preemptive;
+import static com.jayway.restassured.RestAssured.given;
 
 import java.util.ArrayList;
 import java.util.List;
 
 import com.frameworkium.config.SystemProperty;
 import com.frameworkium.jira.Config;
-import com.jayway.restassured.RestAssured;
-import com.jayway.restassured.authentication.AuthenticationScheme;
 import com.jayway.restassured.path.json.JsonPath;
 
 public class SearchExecutions {
 
-    private final static AuthenticationScheme auth = preemptive().basic(Config.jiraUsername, Config.jiraPassword);
     private final static String zapiURI = JIRA_URL.getValue() + Config.zapiRestURI;
 
     private final JsonPath jsonPath;
 
     public SearchExecutions(final String query) {
-        RestAssured.baseURI = zapiURI;
-        RestAssured.authentication = auth;
 
-        jsonPath = get(String.format("zql/executeSearch?zqlQuery=%s", query)).andReturn().jsonPath();
+        jsonPath = given()
+                .auth(). preemptive().basic(Config.jiraUsername, Config.jiraPassword)
```

Comment:
Given your private static auth variable in the class, can't you use that rather than creating another auth object for RestAssured?